sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/TransposeWindowSuite.scala

```diff
@@ -160,4 +160,18 @@ class TransposeWindowSuite extends PlanTest {
     comparePlans(optimized, correctAnswer.analyze)
   }
 
+  test("two windows with overlapping project/order by lists") {
```

Comment:
This tests pass with the latest master branch.

Reply:
modified the test to reflect the bug better. Now it fails at the latest master